ggml/src/ggml-sycl/cpy.cpp

```diff
@@ -617,8 +617,20 @@ static void ggml_cpy_i32_i32_sycl(const char * cx, char * cdst, const int ne, co
 
 void ggml_sycl_cpy(ggml_backend_sycl_context & ctx, const ggml_tensor * src0, const ggml_tensor * src1) try {
     // Unlike other operators ggml_sycl_cpy takes 2 distinct tensors instead of a dst ggml_tensor and rely on its src field
-    scope_op_debug_print scope_dbg_print(__func__, src1, /*num_src=*/0,
-                                         std::string(" src0 type=") + ggml_type_name(src0->type));
+    std::stringstream dbg_suffix;
+    if (UNLIKELY(g_ggml_sycl_debug)) {
+        dbg_suffix << "\tsrc0='" << src0->name<<"'"
```

Comment:
At the time I thought about printing the whole `src0` tensor but I intentionally kept it small as the dimensions are assumed to be the same as `src1`. I thought printing the type was enough but we need to add the name as well.
Printing the whole tensor is fine but it would be best to change `debug_print_tensor` so that it returns a string, that would avoid the duplication.

Reply:
PR has been updated now the `debug_print_tensor` has been changed to `debug_get_tensor_str` and replaced in all relevant places